shopfloor/services/location_content_transfer.py

```diff
@@ -253,9 +253,9 @@ def _find_location_to_work_from(self):
             [
                 ("picking_type_id", "in", self.picking_types.ids),
                 ("state", "=", "assigned"),
-                ("printed", "=", False),
+                ("user_id", "in", (False, self.env.user.id)),
```

Comment:
I'm wondering if it's not a use case for te `?=` operator?
```suggestion
                ("user_id", "?=", self.env.user.id),
```

Reply:
No, you cannot do that
`'=?' is a short-circuit that makes the term TRUE if right is None or False`